### PR TITLE
datamash: add page

### PR DESCRIPTION
**Pull request by nmaggioni**
- [X] The page (if new), does not already exist in the repo.

- [X] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [X] The page has 8 or fewer examples.

- [X] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [X] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/linux/datamash.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# datamash
+
+> GNU datamash performs basic numeric, textual and statistical operations on input textual data files.
```
Comment (agnivade) [inline pages/linux/datamash.md]: There's no need to mention that name of the tool again in the heading. We can just say - `Tool to perform basic ..`

File: pages/linux/datamash.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+- Get max, min, mean and median of a single column of numbers:
+
+`seq 3 | datamash max 1 min 1 mean 1 median 1`
+
+- Get the mean of a single column of float numbers (floats must use `,` instead of `.`):
```
Comment (agnivade) [inline pages/linux/datamash.md]: Change from backticks to double quotes. We usually use that for commands.

File: pages/linux/datamash.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+`echo -e '1.0\n2.5\n3.1\n4.3\n5.6\n5.7' | tr '.' ',' | datamash mean 1`
+
+- Get the mean of a single column of numbers with a given decimal precision:
+
+`echo -e '1\n2\n3\n4\n5\n5' | datamash -R {{number_of_decimals_wanted}} mean 1`
+
+- Get the mean of a single column of numbers ignoring `Na` and `NaN` (literal) strings:
```
Comment (agnivade) [inline pages/linux/datamash.md]: Same as before.

**Comment by nmaggioni** [comment]
Done!

**Pull request merged**